### PR TITLE
fix(cta): arrow/icon should never be on a line by itself

### LINKED ISSUE #238: [bug] [pfe-cta] explore preventing the arrow icon in the default CTA from dropping to a line by itself
**Problem:** Having the arrow break onto a line by itself makes it feel disconnected from the link and little broken. 

**Context:** In working on the nav and homepage updates for summit, we have solved this temporarily with larger layout changes.

**Idea:** I wonder if there is a solution that treats the arrow like a letter thats part of the last word. That way it would only break to another line whenever that last word naturally wraps. 

**Examples:**
_An example where the arrow makes the whole card look a bit off._
Screen Shot 2022-04-26 at 3 37 44 PM

_The left CTA is how this currently behaves. The right CTA is what the solution above might look like._ 
Screen Shot 2022-04-26 at 3 38 22 PM

### PR DESCRIPTION
**Pull request by nikkimk**
Fixes #238
## What I did

1. made static icon size
2. positioned icon inline instead of flex

## Notes to Reviewers
I think this addresses what we were talking about. If not, I'm happy to change it.

### CODE REVIEW

File: .changeset/little-hairs-rhyme.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+"@rhds/elements": patch
+---
+
+`<rh-cta>`: made static icon size and positioned icon inline instead of flex
```
Comment (bennypowers) [inline .changeset/little-hairs-rhyme.md:5]: changesets should describe user-facing improvements, not internal code changes. This prose could be a commit message, or a PR title, but shouldn't be in the changelog

**Review by bennypowers — CHANGES_REQUESTED** [review]
Is this a dupe of #835 ?

PTAL @zeroedin

File: .changeset/little-hairs-rhyme.md (hunk 1 above)
Comment (bennypowers) [reply .changeset/little-hairs-rhyme.md:5]: Could you also please update the PR title? I couldn't figure out which problem this is coming to address

Comment (bennypowers) [inline .changeset/little-hairs-rhyme.md:5]:
```suggestion
`<rh-cta>`: ensured that arrow in default CTAs is aligned to the end of the last line of text
```

**Review by bennypowers — CHANGES_REQUESTED** [review]

File: .changeset/little-hairs-rhyme.md (hunk 1 above)
Comment (bennypowers) [reply .changeset/little-hairs-rhyme.md:5]: very clear. thanks! This is #238

**Review by marionnegp — COMMENTED** [review]
I'm still seeing the arrow flow to its own line sometimes.
Screenshot 2023-03-24 at 9 51 40 AM

Screenshot 2023-03-24 at 9 50 38 AM
Screenshot 2023-03-24 at 9 45 17 AM
fc665-d9f2-4791-a5ca-d3f597d5cc7b.png)

**Comment by nikkimk** [comment]
@marionnegp try it now

**Comment by zeroedin** [comment]
> @marionnegp try it now

Does it look like the arrow is resting slightly too far down?  Like `vertical-align: baseline` look more correct?

**Review by zeroedin — APPROVED** [review]
Looney Groovy Toony Moony

File: elements/rh-cta/rh-cta.css [hunk 2]
```diff
@@ -68,14 +68,14 @@
 }
 
 #container.icon {
-  padding-inline-end: calc(0 - var(--_arrow-size));;
+  padding-inline-end: --_arrow-size);
```
Comment (bennypowers) [inline elements/rh-cta/rh-cta.css]:
```suggestion
  padding-inline-end: var(--_arrow-size);
```

**Comment by bennypowers** [comment]
Tests are red ptal

**Review by marionnegp — APPROVED** [review]
Arrow placement looks good to me!

**Review by bennypowers — APPROVED** [review]

**Pull request merged**